test/OpenIddict.Server.Tests/AddServer_RegistersRequiredSingletonDataAttribute.cs

```diff
@@ -0,0 +1,53 @@
+using System.Collections.Generic;
+using System.Reflection;
+using Xunit.Sdk;
+
+namespace OpenIddict.Server.Tests
+{
+    public class AddServer_RegistersRequiredSingletonDataAttribute : DataAttribute
```

Comment:
Same remark, let's used a property in the main tests class instead.

Reply:
Removed.